Modules/Core/Common/include/itkHexahedronCell.h

```diff
@@ -170,13 +173,7 @@ class ITK_TEMPLATE_EXPORT HexahedronCell
                    InterpolationWeightType * weights);
 
 public:
-  HexahedronCell()
-  {
-    for (unsigned int i = 0; i < Self::NumberOfPoints; ++i)
-    {
-      m_PointIds[i] = NumericTraits<PointIdentifier>::max();
-    }
-  }
+  HexahedronCell() { std::fill(std::begin(m_PointIds), std::end(m_PointIds), NumericTraits<PointIdentifier>::max()); }
```

Comment:
Did `clang-format` produce this?

Reply:
> Did clang-format produce this?

Indeed! Time to reconsider our clang-format preferences?  🤔